includes/multi-currency/MultiCurrency.php

```diff
@@ -364,6 +364,11 @@ public function init_settings_pages( $settings_pages ): array {
 			return $settings_pages;
 		}
 
+		// Due to autoloader limitations, we shouldn't initiate MCCY settings if the plugin was just upgraded: https://github.com/Automattic/woocommerce-payments/issues/9676.
```

Comment:
Nit: Should we split this line into two, for readability:
```suggestion
		// Due to autoloader limitations, we shouldn't initiate MCCY settings if the plugin was just upgraded:
		// https://github.com/Automattic/woocommerce-payments/issues/9676.
```